cmd/podman/shared/create.go

```diff
@@ -538,18 +538,29 @@ func ParseCreateOpts(ctx context.Context, c *GenericCLIResults, runtime *libpod.
 		}
 	}
 
-	// env overrides any previous variables
-	cmdlineEnv := c.StringSlice("env")
-	if len(cmdlineEnv) > 0 {
-		parsedEnv, err := envLib.ParseSlice(cmdlineEnv)
-		if err != nil {
-			return nil, err
+	if c.IsSet("env") {
+		// env overrides any previous variables
+		cmdlineEnv := c.StringSlice("env")
```

Comment:
Woah. Is ENV a string slice? That separates automatically on `,` which means we can't have commas in environment variables - that doesn't seem right?

Reply:
Perhaps a different issue, it has been this way for a while.

Reply:
@mheon fwiw, commas in environment variables seems to work just fine.